recipe/deploy/update_code.php

```diff
@@ -46,6 +46,13 @@
 // - clone (if you need the origin repository `.git` dir in your {{release_path}})
 set('update_code_strategy', 'archive');
 
+/**
+ * Forcing checkout to branch/head.
+ * Works only when [`update_code_strategy`](#update_code_strategy) is set to `clone`.
+ *
+ */
+set('force_head', false);
```

Comment:
So what is the difference from the `target`? You can set target to a branch.

Reply:
That's a good question.
We do deployments to production servers based on git tags, but the GIT automation functionality of Statamic requires us to set the remote to a branch.

Reply:
Wow) This is odd. Maybe this can be done in additional task?
```php
task('deploy:set-remote-branch', function () {
     run('git remote set {{branch}}');
});

after('deploy:update_code', 'deploy:set-remote-branch');
```

Reply:
agreed! let's close this PR for now!